CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -535,11 +535,11 @@ public let ATTRIBUTE_NODES: [Node] = [
             name: "Token",
             kind: .token(choices: [
               .token(.identifier),
-              .keyword(text: "private"),
-              .keyword(text: "fileprivate"),
-              .keyword(text: "internal"),
-              .keyword(text: "public"),
-              .keyword(text: "open"),
+              .keyword(.`private`),
+              .keyword(.`fileprivate`),
+              .keyword(.`internal`),
```

Comment:
Do these even need to be backpacked here. IIRC the only case that needs to be backtracked on the call side is `init`.

Same in a couple more places

Reply:
Great point thank you for it! ❤️ Just pushed the corrected version.